examples/experimental/dagster-components/dagster_components_tests/code_locations/custom_sling_location/custom_sling_location/components/debug_sling_component/defs.yml

```diff
@@ -0,0 +1,6 @@
+component_type: ./debug_sling_component
```

Comment:
not a huge fan of this `./` syntax -- feels like a weird mix of filesystem path and not filesystem path (bc it doesn't have the `.py`)

what if this was:

```python
component_type:
  local: debug_sling_component
```

Reply:
Yeah neither of these is particularly satisfactory right now. Will come up with some alts.